### PR TITLE
Register & Login working

### PR DESCRIPTION
**Pull request by MrT3313**
- REGISTER : https://reed-test.herokuapp.com/api/registration/
- SHAPE : username, password1, password2

- LOGIN : https://reed-test.herokuapp.com/api/login/
- SHAPE : username & password

### CODE REVIEW

**Comment by RonnySAlvarado** [comment]
Zoomed with Reed and demonstrated functionality working as intended. Waiting on additional reviewers before merging.

**Comment by louismagdaleno** [comment]
I’ll be reviewing tonight and will merge it in if all is good.

**Review by louismagdaleno — APPROVED** [review]
Verified that this code works. Great job, @MrT3313!  Merging into master.

**Pull request merged**
